cpp-package/src/inc/board_shim.h

```diff
@@ -201,6 +201,7 @@ class BoardShim
     int get_board_data_count ();
     /// get all collected data and flush it from internal buffer
     BrainFlowArray<double, 2> get_board_data ();
+    BrainFlowArray<double, 2> get_board_data (int);
```

Comment:
need to add comment for doxygen like above and it will be displayed in docs properly

Reply:
sure